base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
+\DeclareFontSeriesChangeRule {uc}{eb}   {ebuc} {buc}
+\DeclareFontSeriesChangeRule {uc}{ub}   {ubuc} {buc}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ec}{ul}   {ulec} {}
+\DeclareFontSeriesChangeRule {ec}{el}   {elec} {}
+\DeclareFontSeriesChangeRule {ec}{l}    {lec}  {}
+\DeclareFontSeriesChangeRule {ec}{sl}   {slec} {}
+\DeclareFontSeriesChangeRule {ec}{sb}   {sbec} {bec}
+\DeclareFontSeriesChangeRule {ec}{b}    {bec}  {}
+\DeclareFontSeriesChangeRule {ec}{eb}   {ebec} {bec}
+\DeclareFontSeriesChangeRule {ec}{ub}   {ubec} {bec}
+%    \end{macrocode}
+%
+%    There are a number of font families that implement condensed
+%    series, but often only \texttt{c} and \texttt{bc}. So if we see a with change request
+%    that can't be fulfilled we try to stay with \texttt{c} or \texttt{bc}.
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {c}{ul}    {ulc}  {c}
+\DeclareFontSeriesChangeRule {c}{el}    {elc}  {c}
+\DeclareFontSeriesChangeRule {c}{l}     {lc}   {c}
+\DeclareFontSeriesChangeRule {c}{sl}    {slc}  {c}
+\DeclareFontSeriesChangeRule {c}{sb}    {sbc}  {bc}
+\DeclareFontSeriesChangeRule {c}{b}     {bc}   {}
+\DeclareFontSeriesChangeRule {c}{eb}    {ebc}  {bc}
+\DeclareFontSeriesChangeRule {c}{ub}    {ubc}  {bc}
+\DeclareFontSeriesChangeRule {c}{uc}    {uc}   {c}
+\DeclareFontSeriesChangeRule {c}{ec}    {ec}   {c}
+\DeclareFontSeriesChangeRule {c}{sc}    {sc}   {c}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sc}{ul}   {ulsc} {}
+\DeclareFontSeriesChangeRule {sc}{el}   {elsc} {}
+\DeclareFontSeriesChangeRule {sc}{l}    {lsc}  {}
+\DeclareFontSeriesChangeRule {sc}{sl}   {slsc} {}
+\DeclareFontSeriesChangeRule {sc}{sb}   {sbsc} {bsc}
+\DeclareFontSeriesChangeRule {sc}{b}    {bsc}  {}
+\DeclareFontSeriesChangeRule {sc}{eb}   {ebsc} {bsc}
+\DeclareFontSeriesChangeRule {sc}{ub}   {ubsc} {bsc}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sx}{ul}   {ulsx} {}
+\DeclareFontSeriesChangeRule {sx}{el}   {elsx} {}
+\DeclareFontSeriesChangeRule {sx}{l}    {lsx}  {}
+\DeclareFontSeriesChangeRule {sx}{sl}   {slsx} {}
+\DeclareFontSeriesChangeRule {sx}{sb}   {sbsx} {bsx}
+\DeclareFontSeriesChangeRule {sx}{b}    {bsx}  {}
+\DeclareFontSeriesChangeRule {sx}{eb}   {ebsx} {bsx}
+\DeclareFontSeriesChangeRule {sx}{ub}   {ubsx} {bsx}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {x}{ul}    {ulx}  {}
+\DeclareFontSeriesChangeRule {x}{el}    {elx}  {}
+\DeclareFontSeriesChangeRule {x}{l}     {lx}   {}
+\DeclareFontSeriesChangeRule {x}{sl}    {slx}  {}
+\DeclareFontSeriesChangeRule {x}{sb}    {sbx}  {bx}
+\DeclareFontSeriesChangeRule {x}{b}     {bx}   {}
+\DeclareFontSeriesChangeRule {x}{eb}    {ebx}  {bx}
+\DeclareFontSeriesChangeRule {x}{ub}    {ubx}  {bx}
+\DeclareFontSeriesChangeRule {x}{sx}    {sx}   {}
+\DeclareFontSeriesChangeRule {x}{ex}    {ex}   {}
+\DeclareFontSeriesChangeRule {x}{ux}    {ux}   {}
```

Comment:
```suggestion
```

These rules are superfluous. (If the third argument is equal to the second and the fourth argument is empty, then the same series is selected without the rule.)

Reply:
talking nonsense ... yes those particular ones should go